backend/src/core/enums.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum, unique
+from functools import lru_cache
+from operator import attrgetter
+
+
+@unique
+class BaseEnum(Enum):
+    @classmethod
+    @lru_cache(None)
+    def values(cls):
+        return tuple(map(attrgetter("value"), cls))
+
+
+class GenderTypeEnum(str, BaseEnum):
```

Comment:
Is it better to inherit str in BaseEnum? In this case your enums can be inherited from BaseEnum only.

Reply:
Agreed

Reply:
No this is  BaseEnum class and thanks to this you will be able extend logic in the future